dashboard_test.cfg

```diff
@@ -1,5 +1,4 @@
 [dashboard]
 TEST_DIR=./zeeguu_api/tests/
+LOG_DIR=./
 N=5
-SUBMIT_RESULTS_URL=https://zeeguu.unibe.ch/api/dashboard/submit-test-results
-
```

Comment:
i'm happy that this is not hardcoded anymore :) 
but how come is it not needed?

Reply:
It's still needed, but for testing purposes, I turned the uploading of unit test results off.
This way, the live deployment of the Dashboard would not receive many Travis test results for the same Zeeguu-API version while I was testing the Travis configuration. 
But this can easily be turned back on again by adding the submit-results-url to the config file whenever necessary.